src/balance.js

```diff
@@ -1,17 +1,32 @@
 const { BN } = require('./setup');
 
-async function balanceCurrent (account) {
-  return new BN(await web3.eth.getBalance(account));
+class Tracker {
+  constructor (acc) {
+    this.account = acc;
+    this.balances = [];
+  }
+  async delta () {
+    const current = await this.get();
+    return current.sub(this.balances[this.balances.length - 2]);
+  }
+  async get () {
+    const bal = new BN(await web3.eth.getBalance(this.account));
+    this.balances.push(bal);
+    return bal;
+  }
+}
+
+async function balanceTracker (owner) {
+  const tracker = new Tracker(owner);
+  await tracker.get();
+  return tracker;
 }
 
-async function balanceDifference (account, promiseFunc) {
-  const balanceBefore = new BN(await web3.eth.getBalance(account));
-  await promiseFunc();
-  const balanceAfter = new BN(await web3.eth.getBalance(account));
-  return balanceAfter.sub(balanceBefore);
+async function balanceCurrent (account) {
+  return new BN(await web3.eth.getBalance(account));
 }
 
 module.exports = {
-  current: balanceCurrent,
-  difference: balanceDifference,
+  balanceCurrent,
```

Comment:
Following the convention that we use in other helpers (e.g. `time`), these two should be named `current` and `tracker`, so they can be used as `balance.current` and `balance.tracker`.